### ISSUE TITLE
Crash when placing ME Toggle Bus and level emitter

### ISSUE THREAD

**Issue by EnigmaticaGH** [AppliedEnergistics/Applied-Energistics-2#6026]
### Describe the bug

When placing a ME toggle bus or level emitter, the game crashes (same stack trace)

### How to reproduce the bug

1. Place a ME Level Emitter on one a cable in your network, so that it is sticking out
2. Place a ME Toggle Bus in the same manner (as in, it's not conjoined between 2 cables in your network) on the same space as the level emitter
3. Remove the cable with a wrench, then in no particular order, remove the toggle bus and the level emitter
4. Place a toggle bus or level emitter where the cable and other items have been removed
5. ???
6. Crash

### Expected behavior

The game to not crash

### Additional details

Minecraft 1.18.1
Fabric 0.12.12
Applied Energistics 2 v.10.0.0
Other mods and info in the crash log link

### Which minecraft version are you using?

1.18

### On which mod loaders does it happen?

Fabric

### Crash log

https://crashy.net/yqpSfYaLsJfPpxaoCBCT

**Comment by shartte** [comment]
```

java.lang.NullPointerException: Cannot invoke "appeng.api.networking.IGridNode.getGrid()" because the return value of "appeng.parts.automation.StorageLevelEmitterPart.getGridNode()" is null
	at Not Enough Crashes deobfuscated stack trace.(1.18.1+build.22)
	at appeng.parts.automation.StorageLevelEmitterPart$1.onStackChange(StorageLevelEmitterPart.java:100)
	at appeng.me.service.StorageService.onServerEndTick(StorageService.java:90)
	at appeng.me.Grid.onServerEndTick(Grid.java:261)
	at appeng.hooks.ticking.TickHandler.onServerTickEnd(TickHandler.java:292)
	at appeng.hooks.ticking.TickHandler.lambda$init$1(TickHandler.java:93)
	at net.fabricmc.fabric.api.event.lifecycle.v1.ServerTickEvents.lambda$static$2(ServerTickEvents.java:62)
	at net.minecraft.server.MinecraftServer.handler$bdl000$onEndTick(MinecraftServer:4293)
	at net.minecraft.server.MinecraftServer.tick(MinecraftServer:861)
	at net.minecraft.server.integrated.IntegratedServer.tick(IntegratedServer:99)
	at net.minecraft.server.MinecraftServer.runServer(MinecraftServer:684)
	at net.minecraft.server.MinecraftServer.method_29739(MinecraftServer:270)
	at java.lang.Thread.run(Unknown Source)
```

**Comment by Technici4n** [comment]
I managed to reproduce this once (with an import bus without acceleration cards constantly pulling from a creative tank, to trigger the change notification) out of the 100 times or so I've tried... :/

**Comment by shartte** [comment]
Same here. I got it on the first or second try, but then hundreds of tries later, I could not reproduce it anymore.

@EnigmaticaGH Is this actually something you can reproduce on your end or was this a one-off?

**Comment by EnigmaticaGH** [comment]
> Same here. I got it on the first or second try, but then hundreds of tries later, I could not reproduce it anymore.
> 
> @EnigmaticaGH Is this actually something you can reproduce on your end or was this a one-off?

I was able to repro it in a very explicit manner. Sorry for such a late reply, i didn't check Github for a while

**Comment by Technici4n** [comment]
It might have something to do with your modpack. If you can reproduce it reliably with AE2 and fabric alone, please let us know.
Otherwise we will have to close the issue until someone else can provide reproduction steps. :/

**Comment by Technici4n** [comment]
Fixed by #6144.
